wrappers/winuser.go

```diff
@@ -70,6 +70,25 @@ const (
 	SW_FORCEMINIMIZE   = 11
 )
 
+const (
+	WTS_CONSOLE_CONNECT        = 0x1
+	WTS_CONSOLE_DISCONNECT     = 0x2
+	WTS_REMOTE_CONNECT         = 0x3
+	WTS_REMOTE_DISCONNECT      = 0x4
+	WTS_SESSION_LOGON          = 0x5
+	WTS_SESSION_LOGOFF         = 0x6
+	WTS_SESSION_LOCK           = 0x7
+	WTS_SESSION_UNLOCK         = 0x8
+	WTS_SESSION_REMOTE_CONTROL = 0x9
+	WTS_SESSION_CREATE         = 0xA
+	WTS_SESSION_TERMINATE      = 0xB
+)
+
+type WTSSESSION_NOTIFICATION struct {
+	Size      uint32
+	SessionId uint32
+}
```

Comment:
[Nitpick] This structure should come before the constants to match the order in winuser.h.